src/history/HistoryItem.tsx

```diff
@@ -21,16 +20,18 @@ const HistoryItem: FC<TProps> = ({ data: { points, location, creationDate } }) =
         <Icon color="blueGreen" name={isNegative ? 'Gift' : 'Point'} size={20} />
       </Styled.HistoryIcon>
 
-      <View>
+      <Styled.InfoView>
         <Typography fontStyle="bold" size="small">
-          {t(`PROFILE.HISTORY.POINTS_${isNegative ? 'NEGATIVE' : 'POSITIVE'}`)}
+          {t(`PROFILE.HISTORY.POINTS_TITLE`, {
+            title,
+          })}
         </Typography>
         <Typography size="xsmall">{location}</Typography>
-      </View>
+      </Styled.InfoView>
 
       <Styled.Points>
         <Typography color={isNegative ? 'red' : 'secondary'} fontStyle="bold" size="small">
-          {t('PROFILE.HISTORY.POINTS', { points: isNegative ? points : `+${points}` })}
+          {t(`PROFILE.HISTORY.${points > 1 ? 'POINTS' : 'POINT'}`, { points: isNegative ? points : `+${points}` })}
```

Comment:
You could also make use of the plural system provided by i18next: See https://www.i18next.com/translation-function/plurals#singular-plural